### ISSUE TITLE
Audio sliders only go to 0dB max

### ISSUE THREAD

**Issue by sporetec** [dvangennip/obs_web_remote#36]
In the webinterface you can only set a max of 0dB gain while in the advanced audio properties it goes way beyond would be great to implement this. Also if a gain is above 0 the sliders actually go out of their box while still showing 0dB as their value.

**Comment by dvangennip** [comment]
Good catch about the sliders not sticking to their box. An overall fix would be to allow sliders to go above 0 dB, so the 0 dB setting sits a little further down.
